network.env.dist

```diff
@@ -67,65 +71,96 @@ ZONE_2_0_DOM_URL=ws://127.0.0.1
 ZONE_2_1_DOM_URL=ws://127.0.0.1
 ZONE_2_2_DOM_URL=ws://127.0.0.1
 
-# Sub websocket urls
+
+# Subordinate websocket urls
+
 PRIME_SUB_URLS=ws://127.0.0.1:8579,ws://127.0.0.1:8581,ws://127.0.0.1:8583
 REGION_0_SUB_URLS=ws://127.0.0.1:8611,ws://127.0.0.1:8643,ws://127.0.0.1:8675
 REGION_1_SUB_URLS=ws://127.0.0.1:8613,ws://127.0.0.1:8645,ws://127.0.0.1:8677
 REGION_2_SUB_URLS=ws://127.0.0.1:8615,ws://127.0.0.1:8647,ws://127.0.0.1:8679
 
-# Slices that are running
+
+# The SLICES variable determines which slices of the network the node will run.
+# The default configuration of 9 slices will run a "global node."
+# Selecting any subset of slices will run either a "multi-slice node" or a "slice node."
+
 SLICES="[0 0],[0 1],[0 2],[1 0],[1 1],[1 2],[2 0],[2 1],[2 2]"
 
-#Boolean Variable Definition
-#Enables or disables http porting
-#Enables or disables websocket porting
+
+# Boolean Variables
+
+# ENABLE_HTTP enables or disables http porting.
+# ENABLE_WS enables or disables websocket porting.
+# ENABLE_ARCHIVE enables or disables running an archive node.
+
 ENABLE_HTTP=true
 ENABLE_WS=true
-ENABLE_UNLOCK=false
 ENABLE_ARCHIVE=false
 
-#Input Variable Definition
-#Network Options include colosseum and garden 
-#http.addr options include 0.0.0.0 and 127.0.0.1 (local host)
-#ws.addr options include 0.0.0.0 and 127.0.0.1 (local host)
-#ws.api options include any blockchain compatible api
+
+# Input Variables
 
 # WARNING: Only allow websocket connections (i.e. WS_ADDR) from a trusted
 # network. At time of writing, no transport security is implemented, but this is
 # a trusted communication channel.
+
+# NETWORK options include colosseum (mainnet), garden (testnet), and local. 
+# You will need to input the correct NONCE to join the P2P network.
+# HTTP_ADDR options include 0.0.0.0 and 127.0.0.1 (local host)
+# WS_ADDR options include 0.0.0.0 and 127.0.0.1 (local host)
+# WS_API and HTTPS_API options include any blockchain compatible API
+# ENABLE_NAT should be turned to true if port forwarding is being used.
+# EXT_IP should be set to the node's external IP if port forwarding is being used. 
+# Turning RUN_BLAKE3 to true allows for BLAKE3 mining to be performed when running a local node. 
```

Comment:
again, I would prefer these to be in-line comments, or 'right above the line' comments

Reply:
Completed in commit 14a3ef6